kroxylicious-docs/docs/_modules/authorization/con-authorization-primitives.adoc

```diff
@@ -0,0 +1,837 @@
+:_mod-docs-content-type: CONCEPT
+
+
+[id='con-authorization-primitives-{context}']
+= Authorization Primitives
+
+Kafka clients send Kafka messages to perform operations on resources in Kafka.
+It is required to know the operations and resources to effectively configure authorization checks.
+This section lists these operations and resources, and then list the mapping between the Kafka messages and the operations and resources.
+
+== Resource type and operations
+
+This table lists the resources types and operations enforced by the authorization filter.
+
+[id='authorization-resource-types-and-operations-{context}']
+.Table of authorization primitives
+|===
+|Resource type|Operations
+
+|Topic
+|READ, WRITE, CREATE, DELETE ALTER, DESCRIBE, DESCRIBE_CONFIGS, ALTER_CONFIGS
+
+|Cluster
+|CREATE, ALTER, DESCRIBE, CLUSTER_ACTION, DESCRIBE_CONFIGS, ALTER_CONFIGS
+
+|===
+
+== Message, operation, and resource mapping
+
+
+The table below lists the kafka messages for which the authorization filter will make authorization enforcements.
```

Comment:
Personally (maybe @PaulRMellor will disagree), I find its good to make clear when you're referring to a specific filter, rather than using "authorization" generically.

```suggestion
The table below lists the kafka messages for which the `Authorization` filter will make authorization enforcements.
```